src/codeu/chat/common/ServerInfo.java

```diff
@@ -1,14 +1,17 @@
 package codeu.chat.common;
 
 import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
are you using these imports anywhere? I recommend using an IDE with an auto-formatted (like eclipse or intellij) that sorts imports and removes unnecessary ones for you